style.css

```diff
@@ -0,0 +1,96 @@
+h1 {
+    text-align: center;
+    margin: 40px;
+}
+body {
+    font-family: "Arial", sans-serif;
+    font-size: 14px;
+    
+}
+.e1fqkh3o5{
+    background-color: white !important;
+}
+
+form {
+    width: 60%;
+    margin: 20px auto;
+    overflow: visible;
+    padding: 25px;
+}
+p {
```

Comment:
I'm guessing it's this that's making *everything* bold, rather than just the things that are bolded in the design (https://www.figma.com/file/xvBWSyzuURvtIaMKAjAPZY/Data-measurements-tool?node-id=0%3A1). It might be this part that needs to be removed, but definitely check!